app/src/components/ARRItem.tsx

```diff
@@ -2,36 +2,37 @@ import * as React from 'react';
 import Card from 'react-bootstrap/Card';
 
 interface ARRItemProps {
-  drizzle: any;
-  drizzleState: any;
   id: number;
+  contracts: any;
+  accounts: Array<string>;
+}
+
+type ARRItemType = {
```

Comment:
I think we should prefer interfaces everywhere instead of types for props/state.